python/tvm/relay/backend/contrib/tidl.py

```diff
@@ -1000,15 +980,22 @@ def tidl_import_dense(self, this_node):
         import_lib_dense(num_innodes, num_outnodes, ctypes.c_void_p(weights_array.ctypes.data))
         return True
 
-    def tidl_import_mul(self, this_node):

```

Comment:
Is tidl_import_mul supposed to be deleted?

Reply:
Yes, "multiply" is not supported by this version of TIDL.